### DISCUSSION TITLE
Error while creating the IPA

### DISCUSSION THREAD

**Discussion by KSNOFF** [codemagic-ci-cd/codemagic-docs#2418] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

Hello, there were no errors with this project in codemagic before.
The error occurred after I tried to fix the problems of my archive that apple informed me in an email:
ITMS-90683: Missing purpose string in Info.plist
ITMS-90078: Missing Push Notification Entitlement

What did I do:
1. In Xcode, in the Capabilities section, I added Push Notifications and Background Modes.
2. Added the necessary lines to the info.plist
3. In the App store Connect, in the Certificates, Identifiers & Profiles section, in identified, I edited the app id configuration. Enabled Push notifications and added a certificate
4. Created a distribution Profile file of the Appstore type. it contains Push notifications.
5. I started the creation with a build with a new Profile

### Expected results

I expect my build to be uploaded to the app store connect and I will submit it for a Testflight

### Actual results

After making corrections to the project to eliminate these errors, the following error occurs at the end of the building ios phase:

Building App Store IPA... 964ms
Encountered error while creating the IPA:
error: exportArchive: "Runner.app" requires a provisioning profile with the Push Notifications feature.

### Build id (optional)

65f9762bb31eed376f59e473

**Comment by himesh-cm** [comment]
Hi @KSNOFF,
Please resave the provisioning profile on your Apple developer account, re-upload it in your `app > settings > iOS signing`, and run the build again.

Let me know how it goes.

**Comment by himesh-cm** [comment]
I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
